ts/input/tex/base/BaseMethods.ts

```diff
@@ -1221,16 +1221,21 @@ BaseMethods.Cr = function(parser: TexParser, name: string) {
  */
 BaseMethods.CrLaTeX = function(parser: TexParser, name: string, nobrackets: boolean = false) {
   let n: string;
-  if (!nobrackets && parser.string.charAt(parser.i) === '[') {
-    let dim = parser.GetBrackets(name, '');
-    let [value, unit, ] = ParseUtil.matchDimen(dim);
-    // @test Custom Linebreak
-    if (dim && !value) {
-      // @test Dimension Error
-      throw new TexError('BracketMustBeDimension',
-                          'Bracket argument to %1 must be a dimension', parser.currentCS);
+  if (!nobrackets) {
+    if (parser.string.charAt(parser.i) === '*') {  // The * controls page breaking, so ignore it
+      parser.i++;
+    }
```

Comment:
I assume `GetStar` would not work as it ignores spaces.

Reply:
Yes, the direct check rather than `getStar()` was to not ignore spaces.